content/en/flux/installation/_index.md

```diff
@@ -143,7 +143,7 @@ Various configuration options are available at bootstrap time such as:
 
 * [Installing optional components](configuration/optional-components.md)
 * [Enforcing tenant isolation on shared clusters](configuration/multitenancy.md)
-* [Using workload identity on AWS, Azure and GCP](configuration/workload-identity.md)
+* [Using workload identity on AWS, Azure, and GCP](configuration/workload-identity.md)
```

Comment:
Comma before `and` is only used for independent sentences. In a list of items `and` already fulfills the purpose of a comma.